seismicpro/containers.py

```diff
@@ -274,6 +275,151 @@ def apply(self, func, cols, res_cols=None, axis=None, unpack_args=False, inplace
         return self
 
 
+    # pylint: disable-next=too-many-arguments
+    def load_headers(self, path, headers_names=None, join_on=None, format="fwf", has_header=False, usecols=None,
+                     sep=',', skiprows=0, decimal=None, encoding="UTF-8", keep_all_headers=False, inplace=False,
+                     **kwargs):
+        """Load headers from a file and join them to `self.headers`.
+
+        Parameters:
+        -----------
+        path : str
+            A path to the file with headers.
+        headers_names : array-like of str, optional, defaults to None
+            An array with column names to use as trace header names. If `has_header` is `True`, then `headers`
+            specifies which columns will be loaded from the file.
+        join_on : str, array-like of str or None, optional, defaults to None
+            Column(s) based on which loaded headers will be joined to `self.headers`. If `None`, intersection of
+            headers from `headers` and `self.headers.columns` will be used.
+        format : "fwf" or "csv", optional, defaults to "fwf"
+            Format of the file with headers. Currently, the following options are supported:
+            * "fwf" - fixed-width format,
+            * "csv" - comma-separated values format.
+        has_header : bool, optional, defaults to False
+            Indicate if the first row of the file contains header names or not.
+        usecols : array-like of int or None, optional, defaults to None
+            Columns indices to be selected from the file. Unlike `pandas` loaders, it is allowed to use negative
+            indices. Should be always passed in ascending order and have the same length as `headers` if both passed.
+        sep : str, defaults to ','
+            Separator used in the file. Used only for "csv" `format`.
+       skiprows : int, optional, defaults to 0
+            Number of rows to skip from the beginning of the file.
+        decimal : str, optional, defaults to None
+            Decimal point character. If not provided, it will be inferred from the file. Used only for "fwf" `format`.
+        encoding : str, optional, defaults to "UTF-8"
+            File encoding.
+        keep_all_headers : bool, optional, defaults to False
```

Comment:
I don't like the naming. Why not
```suggestion
        how : 'inner' or 'left', optional, defaults to 'inner'
```
?